pycbc/waveform/bank.py

```diff
@@ -226,10 +218,9 @@ class TemplateBank(object):
     table : WaveformArray
         An instance of a WaveformArray containing all of the information about
         the parameters of the bank.
-    compressed_waveforms : {None, dict}
-        If compressed waveforms are present in the (hdf) file, a dictionary
-        of those waveforms. Keys are the template hashes, values are
-        `CompressedWaveform` instances.
+    has_compressed_waveforms : {False, bool}
+        If compressed waveforms are present in the (hdf) file, save True,
+        else False .
```

Comment:
This is a bit confusing; suggest:
```
has_compressed_waveforms : bool
    True if compressed waveforms are present in the the (hdf) file; False otherwise.
```